globeandmail/sophi-for-wordpress
# Developer Documentation updates.

### Description of the Change
This PR contains changes for update Developer Documentation as suggested in #322 

Closes #8
Closes #322

### Alternate Designs



### Possible Drawbacks



### Verification Process
1. Checkout to this PR branch.
2. Run `npm install && npm run build:docs`
3. Open index.html from `docs` folder and verify the documentation updates.

### Checklist:






### Changelog Entry


> Changed - Developer Documentation updates.

### Credits


Props @iamdharmesh @jeffpaul
